.github/workflows/scanning.yml

```diff
@@ -46,22 +46,22 @@ jobs:
       - name: Set up Maven
         uses: stCarolas/setup-maven@v5
         with:
-          maven-version: 3.8.8
+          maven-version: 3.9.6
```

Comment:
How about using the latest 3.9.x release then (3.9.11)?

Reply:
sure, why not